README.md

```diff
@@ -0,0 +1,31 @@
+# Overview
+
+This charm supports deployment and scaling of a Project Clearwater system.  See http://www.projectclearwater.org for more information on Project Clearwater.
```

Comment:
Also, I think it's worth putting in more of an explanation of what this repo actually is